documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc

```diff
@@ -144,6 +144,28 @@ env:
 <3> The keystore contains the private key for mTLS authentication.
 <4> The password for accessing the keystore.
 
+. If you need a custom `SASL` configuration. For example, to access a cluster in a cloud provider with a custom login module, the `custom` `SASL_MECHANISM` allows for `SASL_CUSTOM_CONFIG`. +
+*NOTE:  Only SASL properties can be set with the `custom` mechanism.*
++
+.Example custom SASL configuration
+[source,shell,subs=+quotes]
+----
+# ....
+env:
+  - name: SASL_MECHANISM # <1>
+    value: custom
```

Comment:
Thinking about it, I'm a little wary of indicating the custom situation like this. Right now this is very clearly expected to be a recognised SASL mechanism. Technically the names of SASL mechanisms are defined by IANA (https://www.iana.org/assignments/sasl-mechanisms/sasl-mechanisms.xhtml), so we'd be trespassing into their namespace if we allowed `custom`.

But I don't think we need to supply `SASL_MECHANISM` _at all_ in the case you want to support: We can make `SASL_MECHANISM` mutually exclusive with `SASL_CUSTOM_CONFIG_JSON`.

Reply:
I was thinking along the same line.